### PR TITLE
Accessibility Bug fixes related to TemplateView, PeoplePickerView, ListItem and AppBarlayout

### PR DESCRIPTION
**Pull request by aishwarya56**
Fixes Accessibility bugs in Demo App and few fixes in PeoplePickerView

The bugs that we are tackling in this PR are
Fixes [AB#9402](https://dev.azure.com/microsoftdesign/e0256516-4c8d-4265-8edc-b3309388a503/_workitems/edit/9402)
Fixes [AB#9419](https://dev.azure.com/microsoftdesign/e0256516-4c8d-4265-8edc-b3309388a503/_workitems/edit/9419)
Fixes [AB#9420](https://dev.azure.com/microsoftdesign/e0256516-4c8d-4265-8edc-b3309388a503/_workitems/edit/9420)
Fixes [AB#9423](https://dev.azure.com/microsoftdesign/e0256516-4c8d-4265-8edc-b3309388a503/_workitems/edit/9423)
Fixes [AB#9436](https://dev.azure.com/microsoftdesign/e0256516-4c8d-4265-8edc-b3309388a503/_workitems/edit/9436)
Fixes [AB#9441](https://dev.azure.com/microsoftdesign/e0256516-4c8d-4265-8edc-b3309388a503/_workitems/edit/9441)
Fixes [AB#9451](https://dev.azure.com/microsoftdesign/e0256516-4c8d-4265-8edc-b3309388a503/_workitems/edit/9451)
Fixes [AB#9481](https://dev.azure.com/microsoftdesign/e0256516-4c8d-4265-8edc-b3309388a503/_workitems/edit/9481)

### CODE REVIEW

File: FluentUI.Demo/src/main/res/layout/activity_app_bar_layout.xml [hunk 1]
```diff
@@ -9,4 +9,6 @@
     android:id="@+id/app_bar_layout_list"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:importantForAccessibility="no"
+    android:background="?attr/fluentuiBackgroundColor"
```
Comment (mnkshukl) [inline FluentUI.Demo/src/main/res/layout/activity_app_bar_layout.xml:13]: I think it was decided not to use attr directly in demo after modularization ?

File: fluentui_others/src/main/java/com/microsoft/fluentui/peoplepicker/PeoplePickerView.kt [hunk 2]
```diff
@@ -292,6 +293,14 @@ class PeoplePickerView : TemplateView {
         peoplePickerTextViewAdapter?.showSearchDirectoryButton = showSearchDirectoryButton
 
         updateHintVisibility()
+        if (context.isAccessibilityEnabled) {
```
Comment (mnkshukl) [inline fluentui_others/src/main/java/com/microsoft/fluentui/peoplepicker/PeoplePickerView.kt:296]: is this required ?

Comment (aishwarya56) [reply fluentui_others/src/main/java/com/microsoft/fluentui/peoplepicker/PeoplePickerView.kt:296]: Yes, we do not want to have keyboard focus on these when not in talkback mode as we do not have any action on these.
But in talkback, these should be announced

File: FluentUI.Demo/src/main/res/layout/activity_app_bar_layout.xml (hunk 1 above)
Comment (aishwarya56) [reply FluentUI.Demo/src/main/res/layout/activity_app_bar_layout.xml:13]: It is used at many places in the demo app as the base theme is Fluent Theme.

As per my understanding the discussion was based on the fact that Demo app theme should not extend FluentTheme 

These all should be changed once, we switch to another theme at once, Or i can raise a PR changing this for now. But it still will be used in other places in Demo app

Comment (mnkshukl) [reply FluentUI.Demo/src/main/res/layout/activity_app_bar_layout.xml:13]: Let's not introduce  one more attr to be replaced , a New color should work fine

**Review by mnkshukl — APPROVED** [review]
replace attr

**Pull request merged**